query-engine/js-connectors/js/neon-driver-adapter/src/neon.ts

```diff
@@ -140,5 +138,11 @@ export class NeonHTTPAdapter extends NeonQueryable implements Connector {
     )
   }
 
-  async close() {}
+  async connect() {
+    debug('[js::connect]')
+  }
+
+  async disconnect() {
+    debug('[js::disconnect]')
+  }
```

Comment:
One would be inclined to run `await this.pool.end()` upon disconnection, releasing the connection pool resources.
This operation can only be done once per pool, or it would trigger an error. However, one can freely alternate `await prisma.$connect()` with `await prisma.$disconnect()`, so we can't release the pool on `disconnect()`.

Would the Node.js process remain hanging due to pending database connections? No, they would be automatically released a few milliseconds after the last query execution.